.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+# Barebones template to ensure pipeline is triggered
```

Comment:
Can this file be in the kfp plugin directory?
At the minimum, this file can include a gitlab file in kfp subdir.